pkg/web/templates/traveladdress.html

```diff
@@ -0,0 +1,41 @@
+{{ template "base" . }}
+{{ define "styles" }}
+  <link rel="stylesheet" href="/static/styles/traveladdress.css" />
+{{ end }}
+
+{{ define "content" }}
+
+<section class="mx-8 py-12">
+  <h1 class="font-semibold text-2xl md:text-3xl text-center text-balance">
+    Travel Address Encoder/Decoder
+  </h1>
+
+  <section class="my-2 md:mx-8">
+    <p class="mb-4 md:text-lg text-center text-balance">
+      Encode and Decode Travel Addresses as defined in the
+      <a href="https://gitlab.com/OpenVASP/travel-rule-protocol" target="_blank" class="underline hover:text-[#55ACD8]">OpenVASP specifications</a>.
+    </p>
+
+    <form id="traddr-encoder-form" class="mx-auto" action="/v1/utilities/travel-address/encode" method="POST">
+      <textarea id="input-content" name="decoded" nrows="3" placeholder="beneficiary.com/trisa?t=i"></textarea>
```

Comment:
It may be better to display an example above the text box as it may be easy for users to overlook the placeholder.

Reply:
Good idea, but maybe something for later? I feel like if I mess with styles/html I'll get into a quagmire and I needed to timebox this effort.